package.json

```diff
@@ -138,32 +108,51 @@
         "@semantic-release/git",
         {
           "assets": [
+            "./package.json",
             "./CHANGELOG.md",
-            "package.json"
+            "./README.md",
+            "./docs/api.md"
           ]
         }
       ],
       "@semantic-release/github"
     ]
   },
-  "commitlint": {
-    "extends": [
-      "@commitlint/config-conventional"
-    ]
-  },
   "publishConfig": {
     "access": "public"
   },
-  "contributors": [
-    {
-      "name": "Dale Sande",
-      "email": "dale.sande@gmail.com"
-    }
-  ],
   "keywords": [
     "alaska airlines",
     "auro",
     "design system",
     "web components"
-  ]
+  ],
+  "scripts": {
+    "build": "npm-run-all build:sass sass:render scssLint dist:js bundler build:demoScripts postinstall build:api types build:docs",
```

Comment:
You will want to move `build:demoScripts` to the end of the sequence. I noticed that the minified files get overridden when the docs are built out during `npm run:build`. This happened to me when working on button.